deployments/rbac/rbac.yaml

```diff
@@ -94,6 +94,16 @@ rules:
     - list
     - watch
     - get
+- apiGroups:
+    - cert-manager.io
+  resources:
+    - certificates
+  verbs:
```

Comment:
Does cert-manager handled the lifecycle of Certificates? Without a `delete` will we orphan or leak them over time?

Reply:
They are deleted by cert-manager I think when we delete a VirtualServer, but there could be scenarios where we need to delete them (and indeed we have a function to do so). Good catch, I've added the delete verb here.